Cargo.toml

```diff
@@ -19,9 +19,36 @@ authors = [
     "Tulika Shukla <tulikas20@iitk.ac.in>",
 ]
 edition = "2018"
+exclude = [
+    "docs/**",
+    "*.sh",
+    ".github/**",
+    ".travis.yml",
+    ".circleci/**"
+]
+license="AGPL-3.0-only"
+
+
+[features]
+com = []
+sensors=[]
+analog=[]
+atmega328p=[]
+atmega328p-hal=[]
+atmega328p-core = ["atmega328p","atmega328p-hal"]
+atmega2560p=[]
+atmega2560p-hal=[]
+atmega2560p-core = ["atmega2560p","atmega2560p-hal"]
+atmega328p-com=["atmega328p-core","com"]
+atmega2560p-com=["atmega2560p-core","com"]
+default=["analog","sensors","atmega328p-com","atmega2560p-com"]
```

Comment:
The default should be empty.

Reply:
Once the branch is ready to be merged into main I will remove all the default features so that it is in unison with other parts of the project.